lib/member_page.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class String
+  def tidy
+    gsub(/[[:space:]]+/, ' ').strip
+  end
+end
+
+class MemberPage < Scraped::HTML
+  # TODO: Add contact info
+  field :source do
+    url
+  end
+
+  field :image do
+    noko.at_css('.dep_img img/@src').text.tidy
+  end
+
+  field :name do
+    name_and_id.split('-')[0..-2]
+               .map(&:upcase)
+               .join(' ')
+  end
+
+  field :name__ru do
+    noko.at_css('.dep_info h1').text.tidy
+  end
+
+  field :id do
+    name_and_id.split('-').last
+  end
```

Comment:
Quite a minor point, but it's generally nicer to have the `:id` as the first `field` defined (and, more generally, to arrange the fields in a readable order) as the columns created on Morph will be in the order of definition here.